src/modules/CreateClassForm/components/CreateClassForm.js

```diff
@@ -1,135 +1,109 @@
-import React from 'react';
 import PropTypes from 'prop-types';
-import styled from 'styled-components';
-import { px2em } from 'styles/functions';
+import React from 'react';
 import { Form } from 'react-controlled-form';
+import styled from 'styled-components';
 import { Button, DateSelect, FormField, Label, Select, SubmitButton } from 'modules/common';
+import { px2em } from 'styles/functions';
+
+const toOption = ({ id, name }) => ({ label: name, value: id });
 
 const FormGroup = styled.div`margin-bottom: ${px2em(10)};`;
 
-function CreateClassForm({ onSubmit }) {
+const YesNoGroup = ({ fieldId, label, placeholder }) => (
```

Comment:
Should probably make a card for creating a Checkbox component to replace this down the line.